web-module/src/main/webapp/resources/structEditJS/dateTime/celements-date-time-field.js

```diff
@@ -249,60 +256,95 @@
     #inputTimeField;
     #dateTimePickerFactory;
 
-    #updateHiddenFromVisibleBind;
-
     constructor(dateTimeComponent) {
       this.#dateTimeComponent = dateTimeComponent;
       this.#dateTimePickerFactory = new CelementsDateTimePickerFactory();
-      this.#updateHiddenFromVisibleBind = this.#updateHiddenFromVisible.bind(this);
     }
 
     initFields() {
-      if (!this.#inputDateField) {
-        this.#initDateField();
+      const pickerConfig = this.#collectPickerConfig();
+      if (!this.#inputDateField && this.#dateTimeComponent.hasDateField()) {
+        this.#initDateField(pickerConfig);
       }
       if (!this.#inputTimeField && this.#dateTimeComponent.hasTimeField()) {
-        this.#initTimeField();
+        this.#initTimeField(pickerConfig);
       }
+      this.#setMinMax(this.#dateTimeComponent.siblings);
     }
 
-    #initDateField() {
+    #initDateField(pickerConfig) {
       try {
         this.#inputDateField = this.#dateTimePickerFactory
-          .createDatePickerField(this.#dateTimeComponent.datePart, {
-            defaultDate: this.#dateTimeComponent.defaultPickerDate || false
-          });
-        this.#inputDateField.celStopObserving(this.#inputDateField.FIELD_CHANGED,
-          this.#updateHiddenFromVisibleBind);
-        this.#inputDateField.celObserve(this.#inputDateField.FIELD_CHANGED,
-          this.#updateHiddenFromVisibleBind);
-        this.#inputDateField.setValue(this.#dateTimeComponent.date || '');
+            .createDatePickerField(this.#dateTimeComponent.datePart, pickerConfig);
+        this.#inputDateField.celObserve(EVENT_FIELD_CHANGED, this.#onDateTimeChange.bind(this));
+        this.#inputDateField.value = this.#dateTimeComponent.date || '';
       } catch (exp) {
         console.error('#initDateField: failed to initialize dateField.', this.#dateTimeComponent, exp);
       }
     }
 
-    #initTimeField() {
+    #initTimeField(pickerConfig) {
       try {
         this.#inputTimeField = this.#dateTimePickerFactory
-          .createTimePickerField(this.#dateTimeComponent.timePart, {
-            defaultTime: this.#dateTimeComponent.defaultPickerTime || false,
-            step: this.#dateTimeComponent.timeStep
-          });
-        this.#inputTimeField.celStopObserving(this.#inputTimeField.FIELD_CHANGED,
-          this.#updateHiddenFromVisibleBind);
-        this.#inputTimeField.celObserve(this.#inputTimeField.FIELD_CHANGED,
-          this.#updateHiddenFromVisibleBind);
-        this.#inputTimeField.setValue(this.#dateTimeComponent.time || '');
+            .createTimePickerField(this.#dateTimeComponent.timePart, pickerConfig);
+        this.#inputTimeField.celObserve(EVENT_FIELD_CHANGED, this.#onDateTimeChange.bind(this));

```

Comment:
Removing `celStopObserving` breaks 'only Once' idempotency if `initTimeField` is executed multiple times.